src/jsdoc/react/react-parser.ts

```diff
@@ -58,6 +58,8 @@ function fromReactDocs({ description, displayName, props, methods }, filePath):
 }
 
 function stringifyType(prop: { name: string; value?: any; raw?: string }): string {
+  if (!prop) return '?'; // TODO!
```

Comment:
@KutnerUri , are you sure this could happen? (it wasn't happening before).

Reply:
happened to me before I added `tsType`, so it shouldn't happen anymore.
I think it's better to have a default fallback instead of an exception.